api/Window.json

```diff
@@ -2011,20 +2011,36 @@
               "partial_implementation": true,
               "notes": "The event handler property (<code>ongamepadconnected</code>) is not supported; see <a href='https://crbug.com/1192878'>bug 1192878</a>."
             },
-            "firefox": {
-              "version_added": "29"
-            },
-            "firefox_android": {
-              "version_added": "32"
-            },
+            "firefox": [
+              {
+                "version_added": "89"
+              },
+              {
+                "version_added": "29",
+                "partial_implementation": true,
+                "notes": "The event handler property (<code>ongamepadconnected</code>) is not supported; see <a href='https://bugzil.la/1697093'>bug 1697093</a>."
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "89"
+              },
+              {
+                "version_added": "32",
+                "partial_implementation": true,
+                "notes": "The event handler property (<code>ongamepadconnected</code>) is not supported; see <a href='https://bugzil.la/1697093'>bug 1697093</a>."
+              }
+            ],
             "ie": {
               "version_added": false
             },
             "oculus": "mirror",
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "10.1"
+              "version_added": "10.1",
+              "partial_implementation": true,
+              "notes": "The event handler property (<code>ongamepadconnected</code>) is not supported; see <a href='https://webkit.org/b/223860'>bug 223860</a>."
```

Comment:
Perhaps this should be using the new `impl_url` property instead to link to the bug?

Reply:
Linking to the bug in the notes makes sense.  `impl_url` is for linking the bug where the feature _is_ implemented, but the bug is related to the notes specifically!

Reply:
```suggestion
              "notes": "The <code>ongamepadconnected</code> event handler property is not supported. See <a href='https://webkit.org/b/223860'>bug 223860</a>."
```